HiP-DataStore/Controllers/ExhibitPagesController.cs

```diff
@@ -48,12 +48,15 @@ public class ExhibitPagesController : Controller
 
         [HttpGet("Pages/ids")]
         [ProducesResponseType(typeof(IReadOnlyCollection<int>), 200)]
-        public IActionResult GetAllIds(ContentStatus? status)
+        public IActionResult GetAllIds(ContentStatus status = ContentStatus.Published)
         {
             if (!ModelState.IsValid)
                 return BadRequest(ModelState);
 
-            return Ok(_entityIndex.AllIds(ResourceType.ExhibitPage, status ?? ContentStatus.Published, User.Identity));
+            if (status == ContentStatus.Deleted && !UserPermissions.IsAllowedToGetDeleted(User.Identity))
+                return Forbid();
```

Comment:
In many locations (like this one), `[ProducesResponseType(403)]` is missing

Reply:
You are right, but this annotation is for swagger and swagger can detect forbid(403) automatically